applications/mp/pom.xml

```diff
@@ -32,7 +32,7 @@
     <description>Parent pom for Helidon MP applications</description>
 
     <properties>
-        <version.plugin.jandex>1.0.6</version.plugin.jandex>
+        <version.plugin.jandex>3.1.1</version.plugin.jandex>
```

Comment:
In case you care they released 3.1.2 a few days ago

Reply:
Progress is relentless. Updated.